src/Form/FastlySettingsForm.php

```diff
@@ -402,6 +423,15 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
     if (!$this->api->validatePurgeToken()) {
       $form_state->setErrorByName('api_key', $this->t('Invalid API token. Make sure the token you are trying has at least <em>global:read</em>, <em>purge_all</em>, and <em>purge_all</em> scopes.'));
     }
+
+    // Validate key length for hashed cache tags.
+    $keyLength = $form_state->getValue('cache_tag_hash_length');
```

Comment:
rather than doing this in the validation hook, place it inline in the form API definition.

Reply:
Modified to suggested.